.travis.yml

```diff
@@ -45,9 +45,20 @@ jobs:
                 - ./dev-tools/vendor/bin/composer-require-checker check composer.json --config-file=.composer-require-checker.json
 
 
+        -
+            stage: Fast Test
```

Comment:
Why not reuse `STANDARD_TEST_JOB` and override `stage` only?

Reply:
I tried in the first commit:
1. https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3250/commits/e565c1b2993c0dcb174a214071bc28049c086513
1. https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/builds/301347812?utm_source=github_status&utm_medium=notification